src/incubator/WheelPicker/index.tsx

```diff
@@ -2,16 +2,17 @@
 // TODO: Support style customization
 // TODO: Support control of visible items
 import _ from 'lodash';
-import React, {useCallback, useRef, useMemo} from 'react';
-import {TextStyle, FlatList, NativeSyntheticEvent, NativeScrollEvent} from 'react-native';
+import React, {useCallback, useRef, useMemo, useEffect} from 'react';
+import {TextStyle, ViewStyle, FlatList, NativeSyntheticEvent, NativeScrollEvent} from 'react-native';
 import Animated from 'react-native-reanimated';
 import {onScrollEvent, useValues} from 'react-native-redash';
 import {Colors} from '../../../src/style';
 import View from '../../components/view';
 import Fader, {FaderPosition} from '../../components/fader';
 import {Constants} from '../../helpers';
-import useMiddleIndex from './helpers/useListMiddleIndex';
 import Item, {ItemProps} from './Item';
+import {Spacings} from 'react-native-ui-lib';
```

Comment:
you "can't" import from `react-native-ui-lib` in source files. 
You can get `Spacings` from this import 
`import {Colors} from '../../../src/style';`

Reply:
Right, auto-import